.ci/static.sh

```diff
@@ -12,9 +15,20 @@ function usage {
 }
 
 if [[ "$COMMAND" == "install" ]]; then
+    conda install jupyter
     pip install codespell flake8 pylint
 elif [[ "$COMMAND" == "run" ]]; then
-    flake8 -v nengo && codespell -q 3 && pylint nengo
+    jupyter-nbconvert \
+        --log-level WARN \
+        --to python \
+        --TemplateExporter.exclude_input_prompt=True \
+        -- **/*.ipynb
+    sed -i -e 's/# $/#/g' -e '/get_ipython()/d' -- docs/**/*.py
```

Comment:
Would you mind adding a comment explaining what this line accomplishes for the shell noobs?

Reply:
Can do :+1:

Reply:
Did.